docs/howto/push.md

```diff
@@ -45,7 +45,15 @@ The output is the files `sqlc` would have sent without the `--dry-run` flag.
 Once you're ready to push, remove the `--dry-run` flag.
 
 ```shell
-sqlc push
+$ sqlc push
+```
+
+### Tags
+
+Each push can be associated with a set of tags. Tags only refer to a single push and are overwritten by subsequent `push` commands.
```

Comment:
```suggestion
You can provide tags to associate with a push, primarily as a convenient reference when using `sqlc verify` with the `against` argument.

Tags only refer to a single push, so if you pass an existing tag to `push` it will overwrite the previous reference.
```